tests/test_scaler.py

```diff
@@ -86,9 +86,11 @@ def test_scaler_functionality(self):
         str(sca)
 
         sca.stage()
-        sca.configure()
+        old, new = sca.configure({})
```

Comment:
done here, but Travis is stalling on the scaler tests, so there is probably a typo lurking: https://github.com/danielballan/ophyd/tree/new_configure